autoload/codefmt/googlejava.vim

```diff
@@ -0,0 +1,72 @@
+" Copyright 2017 Google Inc. All rights reserved.
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+
+let s:plugin = maktaba#plugin#Get('codefmt')
+
+""
+" @private
+" Formatter: google-java-format
+function! codefmt#googlejava#GetFormatter() abort
+  let l:formatter = {
+      \ 'name': 'google-java-format',
+      \ 'setup_instructions': 'Install google-java formatter ' .
+          \ "(https://github.com/google/google-java-format). \n" .
+          \ 'Enable with "Glaive codefmt google_java_executable=' .
+          \ '"java -jar /path/to/google-java-format-1.2-all-deps.jar" ' .
```

Comment:
Use placeholder version numbers like /path/to/google-java-format-X.Y-all-deps.jar so we don't get pull reqs bumping this later?